scripts/validation/validate.js

```diff
@@ -15,127 +14,99 @@ import serviceSchema from './service.schema.js';
 const fs = fsApi.promises;
 const __dirname = path.dirname(new URL(import.meta.url).pathname);
 const { expect } = chai;
-const rootPath = path.join(__dirname, '../..');
+
+
+const serviceDeclarationsPath = path.join(__dirname, '../..', config.get('serviceDeclarationsPath'));
 const MIN_DOC_LENGTH = 100;
 
-let serviceDeclarations;
-let modifiedServices;
+
+const args = process.argv.slice(2);
+let schemaOnly = false;
+if (args.includes('--schema-only')) {
+  args.splice(args.indexOf('--schema-only', 1));
+  schemaOnly = true;
+}
+
 (async () => {
-  try {
-    serviceDeclarations = await loadServiceDeclarations(path.join(rootPath, config.get('serviceDeclarationsPath')));
-    modifiedServices = await getModifiedServices();
+  const serviceDeclarations = await loadServiceDeclarations(serviceDeclarationsPath);
 
-    describe('Services validation', async () => {
-      const specifiedServiceId = process.argv.slice(process.argv.indexOf('--serviceId'))[1];
-      const schemaOnly = process.argv.indexOf('--schema-only') != -1;
-      const modifiedOnly = process.argv.indexOf('--modified-only') != -1;
+  const servicesToValidate = args.length ? args : Object.keys(serviceDeclarations);
 
-      const serviceIds = Object.keys(serviceDeclarations);
+  servicesToValidate.forEach(serviceId => {
+    if (!serviceDeclarations.hasOwnProperty(serviceId)) {
+      throw new Error(`Could not find any service with id "${serviceId}"`);
```

Comment:
Because this has the same effect without making the assumption that we're alone to operate and that we are allowed to exit the process 😉 I don't believe `process.exit` should be used.